lib/landrush/command.rb

```diff
@@ -63,7 +67,7 @@ def help; <<-EOS.gsub(/^      /, '')
         set { <host> <ip> | <alias> <host> }
           adds the given host-to-ip or alias-to-hostname mapping.
           existing host ip addresses will be overwritten
-        rm, del { <host> | <alias> }
+        rm, del { <host> | <alias> | --all }
```

Comment:
I think it should be more:

    rm, del [---all] { <host> | <alias> }  

WDYT?